aptos-move/block-executor/src/proptest_types/types.rs

```diff
@@ -620,22 +712,44 @@ impl<V: Into<Vec<u8>> + Arbitrary + Clone + Debug + Eq + Sync + Send> Transactio
     >(
         self,
         universe: &[K],
-        // Are writes and reads module access (same access path).
-        module_access: (bool, bool),
     ) -> MockTransaction<KeyType<K>, E> {
-        let is_module_read = |_| -> bool { module_access.1 };
-        let is_module_write = |_| -> bool { module_access.0 };
-        let is_delta = |_, _: &V| -> Option<DeltaOp> { None };
-        // Module deletion isn't allowed.
-        let allow_deletes = !(module_access.0 || module_access.1);
+        self.new_mock_write_txn(universe, true, None)
+    }
 
-        self.new_mock_write_txn(
-            universe,
-            &is_module_read,
-            &is_module_write,
-            &is_delta,
-            allow_deletes,
-        )
+    pub(crate) fn materialize_modules<
+        K: Clone + Hash + Debug + Eq + Ord,
+        E: Send + Sync + Debug + Clone + TransactionEvent,
+    >(
+        self,
+        universe: &[K],
+    ) -> MockTransaction<KeyType<K>, E> {
+        let universe_len = universe.len();
+        assert_ge!(universe_len, 3, "Universe must have size >= 3");
```

Comment:
Why 3? Makes sense for groups, but here?

Reply:
good catch, no reason